imageio_ffmpeg/_io.py

```diff
@@ -260,6 +260,8 @@ def read_frames(
             else:  # stop_policy == "kill"
                 # Just kill it
                 p.kill()
+        # close stderr at the very end
+        p.stderr.close()
```

Comment:
Quick disable to test that this is indeed causing CI to fail
```suggestion
        #p.stderr.close()
```